docs/scheduler/filter.md

```diff
@@ -28,9 +28,10 @@ You can choose the filter(s) you want to use with the `--filter` flag of `swarm
 Constraints are key/value pairs associated to particular nodes. You can see them
 as *node tags*.
 
-When creating a container, the user can select a subset of nodes that should be
-considered for scheduling by specifying one or more sets of matching key/value
-pairs. This approach has several practical use cases such as:
+When creating a container or building an image, the user can select a subset of
```

Comment:
@pwnall Thank you for the contribution.  The structure can be tightened by doing this:

```markdown
When creating a container or building an image, you can identify a subset of
nodes to consider for scheduling. This approach has several practical use cases such as:

* Selecting specific host properties (such as `storage=ssd`, in order to schedule
  containers on specific hardware).
* Tagging nodes based on their physical location (`region=us-east`, to force
  containers to run on a given location).
* Logical cluster partitioning (`environment=production`, to split a cluster into
  sub-clusters with different properties).

To specify a subset of key/value pairs for a node, pass a list of `--label`
options at docker startup time. For instance, let's start `node-1` with the
`storage=ssd` label:
```

Which yields this: 

----

When creating a container or building an image, you can identify a subset of
nodes to consider for scheduling. This approach has several practical use cases such as:

* Selecting specific host properties (such as `storage=ssd`, in order to schedule
  containers on specific hardware).
* Tagging nodes based on their physical location (`region=us-east`, to force
  containers to run on a given location).
* Logical cluster partitioning (`environment=production`, to split a cluster into
  sub-clusters with different properties).

To specify a subset of key/value pairs for a node, pass a list of `--label`
options at docker startup time. For instance, let's start `node-1` with the
`storage=ssd` label: